include/experimental/__p0009_bits/extents.hpp

```diff
@@ -167,7 +167,7 @@ template <> struct index_sequence_scan_impl<0> {
 // all static values.
 
 template <class T, size_t N> struct possibly_empty_array {
-  T vals[N];
+  T vals[N]{};
```

Comment:
@crtrott I think this is harmless and might actually be _more_ conforming (it certainly looks more like the exposition-only `array` in the `extents` wording), but please do take a look; thanks!